ui/apps/platform/scripts/cypress-component.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+artifacts_dir="${TEST_RESULTS_OUTPUT_DIR:-cypress/test-results}/artifacts"
+export CYPRESS_VIDEOS_FOLDER="${artifacts_dir}/videos"
+export CYPRESS_SCREENSHOTS_FOLDER="${artifacts_dir}/screenshots"
+
+DEBUG="*" NO_COLOR=1 cypress "$@" 2> /dev/null
```

Comment:
It feels odd to discard stderr (though I see this in the other script). I'd save it.
```suggestion
DEBUG="*" NO_COLOR=1 cypress "$@" 2> "${ARTIFACT_DIR:-/tmp}/cypress-err.txt"
```